std/functional.d

```diff
@@ -60,7 +60,8 @@ Distributed under the Boost Software License, Version 1.0.
 */
 module std.functional;
 
-import std.meta, std.traits;
+import std.meta; // AliasSeq, Reverse
+import std.traits; // isCallable, Parameters
```

Comment:
`AliasSeq` is only called from one unittest block and `Parameters` from four templates, but if I make these two symbols into local selective imports and have only the two remaining symbols selectively imported at module scope, I started getting template instantiation errors.  Not sure what's going on, so just moved those two up here.